lib/ansible/modules/monitoring/zabbix/zabbix_host_triggers.py

```diff
@@ -0,0 +1,272 @@
+                    'supported_by': 'community'}
+
+RETURN = '''
+---
+output:
+    description: Host Zabbix Triggers in Json format
+    returned: On success
+    type: complex
+    contains:
+            triggers_ok:
+                - comments: Raise an alert whenever there is no data received from the agent during
+                    the given interval
+                  description: 'Status {HOSTNAME}: server is unreachable'
+                  error: ''
+                  expression: "{13665}=1"
+                  flags: '0'
+                  lastchange: '1556112352'
+                  priority: '4'
+                  state: '0'
+                  status: '0'
+                  templateid: '13619'
+                  triggerid: '14038'
+                  type: '0'
+                  url: ''
+                  value: '0'
+            triggers_problem:
+                - comments: ''
+                  description: 'Process: NTPd is down'
+                  error: ''
+                  expression: "{13667}=0"
+                  flags: '0'
+                  last_event:
+                    acknowledged: '0'
+                    acknowledges:
+                    - alias: Admin
+                      clock: '1556182269'
+                      message: waiting customer action
+                    clock: '1556182854'
+                    eventid: '630'
+                    value: '1'
+                  lastchange: '1556182854'
+                  priority: '4'
+                  state: '0'
+                  status: '0'
+                  templateid: '13615'
+                  triggerid: '14040'
+                  type: '0'
+                  url: ''
+                  value: '1'
+'''
+
+DOCUMENTATION = '''
+---
+module: zabbix_host_triggers
+short_description: Get all triggers about a Zabbix host
+description:
+   - This module allows you to see if a Zabbix host have no active alert to make actions on it.
+     For this case use module Ansible 'fail' to exclude host in trouble.
+   - Length of "triggers_ok" allow if template's triggers exist for Zabbix Host
+version_added: "2.9"
+author:
+    - "Stéphane Travassac (@stravassac)"
+requirements:
+    - "python >= 2.7"
+    - zabbix-api
+options:
+    server_url:
+        description:
+            - Url of Zabbix.
+        required: true
+    login_user:
+        description:
+            - Login of Zabbix.
+        required: true
+    login_password:
+        description:
+            - Password of Zabbix.
+        required: true
+    host_identifier:
+        description:
+            - Identifier of Zabbix Host
+        default: hostname
+        required: true
+    host_id_type:
+        description:
+            - Type of host_identifier
+        choices:
+            - hostname
+            - visible_name
+            - hostid
+        required: false
+        default: hostname
+    trigger_severity:
+        description:
+            - Zabbix severity for search filter
+        default: 3
```

Comment:
a range of available values would be suitable in the description. Also, checking it in the code and then failing implicitly would be imho better than getting python error from zabbix-api module